src/leibniz_compiled.jl

```diff
@@ -0,0 +1,20 @@
+function mainjl()
+    f = fopen(m"rounds.txt", m"r")
+    buf = MallocString(undef, 16)
+    StaticTools.fread!(buf, f)
+    fclose(f)
+    rounds = parse(Int64, buf)
+    free(buf)
+    
+    x = 1.0
+    pi = 1.0
+
+    for i in 2:(rounds + 2)
```

Comment:
Will it still work if this actual computation is a separate function (the same as before) called by `mainjl` which does read/write? That might be prettier / more julian.

Reply:
oh, yes, yes it would, actually we could source the original `jl` file from the `compiled.jl` file

Reply:
@mcabbott done